bert_embeddings/bert_embeddings.ipynb

```diff
@@ -8,85 +8,32 @@
     "This notebook presents deployment of pretrained BERT model that outputs embeddings for given textual sequences as a serverless function. Embeddings are meaningful, contextual representations of text in the form of ndarrays that are used frequently as input to various learning tasks in the field of NLP."
```

Comment:
<p>/# 1) function calls with a lot of parameters should be formatted as if black formatter was applied on them</p><p>/# 2) Don't mix between args and kwargs, if you name one arg, please name the rest of them</p><p><code>fn = code_to_function(</code></p><p><code>	<strong>name=</strong>items["name"],</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;kind=items["spec"]["kind"],</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;filename=items["spec"]["filename"],</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;description=items["description"],</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;categories=items["categories"],</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;labels=items["labels"],</code></p><p><code>&nbsp;&nbsp;&nbsp;&nbsp;requirements=items["spec"]["requirements"],</code></p><p><code>)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/functions/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>